src/USER-MISC/fix_orient_eco.cpp

```diff
@@ -203,6 +201,7 @@ void FixOrientECO::init() {
   neighbor->requests[irequest]->fix = 1;
   neighbor->requests[irequest]->half = 0;
   neighbor->requests[irequest]->full = 1;
+  neighbor->requests[irequest]->ghost = 1;
```

Comment:
It is not necessary to access neighbors of neighbors. the local neighbor list was created and used only because it is significantly shorter than the neighbor list used for the MD potentials. Therefore, this line should be removed! We can't do that here ourselves, it seems.

Reply:
@vmohles you are currently accessing neighbors of neighbors indirectly through the Nbr struct. Otherwise you would not need to do a forward communication to send it to ghost atoms. However, this is not always listing the correct neighbors, since you may have multiple copies of the same "tag" as ghost atoms and you would need to know the index of the atom closest to center atom. 

That means, if you include neighbors of ghost atoms in the neighbor list directly, you could construct the list of struct Nbr elements directly locally without the forward communication. For all atoms "j" that are confirmed neighbors of the short neighbor list, you would also look up and store the neighbors within the cutoff into the corresponding data structure (unless they are already included). You may be getting away with your approach in most cases, since you also store the distance vector to the center atom.

this is **your** branch, and of course you can change it any way your like. since you allowed maintainers to push their changes into your branch, all you have to do is to pull the changes from your github fork into your local fork first, then edit/apply your modifications and push it back.

but since i noticed another change that needs to be made i am apply this now and will push it (**never** commit changes to the makefiles, if you need a customized makefile, make a copy to src/MAKE/MINE and modify that copy and don't add/commit that. those copies will override the version in the other folders).